.github/ISSUE_TEMPLATE/1_feature_request.yml

```diff
@@ -0,0 +1,60 @@
+name: "Product: Feature Request"
+description: "For product team to request a new feature."
+labels: ["refinement", "needs estimate", "enhancement"]
+projects: ["hicommonwealth/33"]
+body:
+  - type: textarea
+    id: feature-description
+    attributes:
+      label: Description
+      description: Describe the requested feature in a few sentences max.
+    validations:
+      required: true
+  - type: input
+    id: owner
+    attributes:
+      label: Project Owner
+      description: Point of contact for questions (if not creator)
+      placeholder: "@sachben91"
```

Comment:
curious. why not something to the effect of enter POC Github handle, rather than defaulting to Sachin. Not blocking, but worth mentioning.

Reply:
It's a placeholder, not a default value -- so Sachin is being used here as an example. It could be anyone.